datalake/src/test/java/com/sequenceiq/datalake/controller/sdx/SdxControllerTest.java

```diff
@@ -190,6 +197,21 @@ void refreshDatahubs() {
         verify(sdxService, times(1)).refreshDataHub(SDX_CLUSTER_NAME, null);
     }
 
+    @Test
+    void validateBackupStorage() {
+        SdxCluster sdxCluster = getValidSdxCluster();
+        sdxCluster.setClusterName(SDX_CLUSTER_NAME);
+        SdxBackupLocationValidationRequest sdxBackupLocationValidationRequest = new SdxBackupLocationValidationRequest(SDX_CLUSTER_NAME);
+        when(storageValidationService.validateBackupStorage(any())).thenReturn(null);
+        when(sdxService.getByNameInAccount(USER_CRN, SDX_CLUSTER_NAME)).thenReturn(sdxCluster);
+
+        ThreadBasedUserCrnProvider.doAs( USER_CRN, () -> sdxController.validateBackupStorage(sdxBackupLocationValidationRequest));
```

Comment:
Please assign the return value (`ValidationResult`) to a local variable and add an assertion that it matches the one returned by `storageValidationService.validateBackupStorage()`.

Reply:
addressed